azure-pipelines.yml

```diff
@@ -151,6 +151,11 @@ jobs:
         - OSX_x64
         - Windows_NT_x64
         - Windows_NT_x86
+      ${{ if eq(variables['Build.DefinitionName'], 'coreclr-outerloop-jitstress-regs-x86') }}:
+        platforms:
+        - Linux_x64
```

Comment:
Unlike with the `jitstress-isas` jobs, OSX is not particularly interesting for the `jitstress-regs` jobs as compared to Linux.